cmd/lakefs/cmd/run.go

```diff
@@ -355,6 +357,29 @@ var runCmd = &cobra.Command{
 			}
 		}()
 
+		isQuickstart, err := cmd.Flags().GetBool(config.QuickstartConfiguration)
+		if err != nil {
+			_, _ = fmt.Fprintf(os.Stderr, "Failed to get command flag %s: %v\n", config.QuickstartConfiguration, err)
+			os.Exit(1)
+		}
+
+		data := bannerData{
+			LocalSetup: localBanner,
+		}
+		if isQuickstart {
+			data.LocalSetup = ""
+			data.Quickstart = quickStartBanner
```

Comment:
This should be done inside the template base on the args we provide.
provide the params we require to render the template - it can include the admin key/secret, the mode and the lakefs version.